dask/dataframe/io/parquet/arrow.py

```diff
@@ -641,6 +845,10 @@ def read_metadata(
 
         # Check if we are using pyarrow.dataset API
         dataset_kwargs = kwargs.get("dataset", {})
+        use_pa_ds = dataset_kwargs.pop("pa_dataset", True) and pa_ds is not None
```

Comment:
Maybe could use the same keyword as in pyarrow, `use_legacy_dataset=True/False`, since it has a bit the same meaning (or otherwise maybe `pyarrow_dataset`, I find `pa_dataset` a bit cryptic)